src/main/reactjs/src/components/contactRequest/ContactRequestFormUtils.tsx

```diff
@@ -113,19 +110,40 @@ export const DisplayContactInfo = () => {
 };
 
 // StepHeading is not shown on mobile devices
-export const StepHeading = ({ step }: { step: string }) => {
+export const StepHeading = ({ step }: { step: ContactRequestFormStep }) => {
   const { t } = useAppTranslation({
     keyPrefix: 'akt.component.contactRequestForm.steps',
   });
+  const translateCommon = useCommonTranslation();
 
+  const [ref, setFocus] = useFocus<HTMLSpanElement>();
   const { isPhone } = useWindowProperties();
+  const numberOfSteps = Object.values(ContactRequestFormStep).filter(
+    (v) => !isNaN(Number(v))
+  ).length;
+  const headingAriaLabel = `${translateCommon(
+    'phase'
+  )} ${step}/${numberOfSteps}: ${t(ContactRequestFormStep[step])}`;
+
+  useEffect(() => {
+    if (!isPhone) {
+      setFocus();
+    }
+  }, [setFocus, isPhone]);
 
   return !isPhone ? (
     <div
       className="contact-request-page__heading"
-      data-testid={`contact-request-page__step-heading-${step}`}
+      data-testid={`contact-request-page__step-heading-${ContactRequestFormStep[step]}`}
     >
-      <H1>{t(step)}</H1>
+      <Typography
```

Comment:
Tässä ei pystytty käyttämään H1?

Reply:
Joo, ei suoraan. React logittaa virheen konsoliin, tuo `ref` passaaminen H1:lle aiheuttaa konsoliin virheviestin `Function components cannot be given refs`.